### PR TITLE
Update chapter info

### PR DESCRIPTION
**Pull request by austingivecamp**
Updating the location, address and URLs for GiveCamp

### CODE REVIEW

File: data/chapters/austin.json [hunk 1]
```diff
@@ -3,20 +3,20 @@
...
-  "longitude": -97.7517582,
+  "latitude": 30.265886,
+  "longitude": -97.749298,
   "img": "austin.jpg",
   "img_attribution": "iStock",
   "events": [
     {
       "start_date": "2016-11-04",
       "end_date": "2016-11-06",
-      "street": "10415 Morado Circle Bld 3 Ste 100",
```
Comment (jayharris) [inline data/chapters/austin.json:14]: Is this intended to be an address change on where the 2016 event was held?

File: data/chapters/austin.json [hunk 2]
```diff
@@ -3,20 +3,20 @@
...
+      "street": "500 W 2nd St",
       "city": "Austin",
       "state": "Texas",
-      "postal_code": 78759,
-      "volunteer_signup_url": null,
-      "non_profit_signup_url": null
+      "postal_code": 78701,
+      "volunteer_signup_url": "https://www.austingivecamp.org/non-profit-form",
+      "non_profit_signup_url": "https://www.austingivecamp.org/volunteer-form"
     }
```
Comment (jayharris) [inline data/chapters/austin.json:20]: Is this to update the 2016 event, or to add the 2019 event? If adding a new event, add another object to the JSON array:

```
"events": [
    {
      "start_date": "2016-11-04",
      "end_date": "2016-11-06",
      "street": "10415 Morado Circle Bld 3 Ste 100",
      "city": "Austin",
      "state": "Texas",
      "postal_code": 78759,
      "volunteer_signup_url": null,
      "non_profit_signup_url": null
    },
    {
      "start_date": "2019-01-01",
      "end_date": "2019-01-03",
      "street": "500 W 2nd St",
      "city": "Austin",
      "state": "Texas",
      "postal_code": 78701,
      "volunteer_signup_url": "https://www.austingivecamp.org/non-profit-form",
      "non_profit_signup_url": "https://www.austingivecamp.org/volunteer-form"
    }
  ]
```

Be sure to change the start and end dates for your 2019 event.

**Comment by jayharris** [comment]
Is this edit to change the location for the 2016 event or to add a 2019 event? As it stands, it is changing the address of where the 2016 event was held. To add a new event, add another object to the `events` array, such as:

```
"events": [
    {
      "start_date": "2016-11-04",
      "end_date": "2016-11-06",
      "street": "10415 Morado Circle Bld 3 Ste 100",
      "city": "Austin",
      "state": "Texas",
      "postal_code": 78759,
      "volunteer_signup_url": null,
      "non_profit_signup_url": null
    },
    {
      "start_date": "2019-01-01",
      "end_date": "2019-01-03",
      "street": "500 W 2nd St",
      "city": "Austin",
      "state": "Texas",
      "postal_code": 78701,
      "volunteer_signup_url": "https://www.austingivecamp.org/non-profit-form",
      "non_profit_signup_url": "https://www.austingivecamp.org/volunteer-form"
    }
  ]
```

File: data/chapters/austin.json (hunk 1 above)
Comment (austingivecamp) [reply data/chapters/austin.json:14]: Yes, it's being moved to a new location.

**Pull request closed without merging**